tokendito/helpers.py

```diff
@@ -422,34 +441,52 @@ def process_okta_aws_app_url(app_url=None):
             " or re-run this application with the --configure flag".format(settings.okta_profile))
         sys.exit(2)
     # Prepare final Okta and AWS app Url
-    url = urlparse(app_url)
+    url = settings.okta_aws_app_url.strip()
```

Comment:
either this should be app_url.strip() or remove the parameter from the function, otherwise it isnt used beside looking for Noneness. Maybe lines 438 to line 451 should be in another function with the settings url as input and a trimmed and validated url as output?